notebooks/jax-vit-mixer/jax-vit-mixer.ipynb

```diff
@@ -0,0 +1,520 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        ![ -e "$model_name"_imagenet2012.npz ] || gsutil cp gs://mixer_models/imagenet1k/"$model_name".npz "$model_name"_imagenet2012.npz
</pre><p>do we really need gsutil for downloading? isit not possible to use standard downloading by link like wget do?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>